library/README.md

```diff
@@ -15,18 +18,27 @@ npm i @howmuchgreen/howmuchcarbon
 ## Usage
 
 ```ts
-import { howMuch } from "@howmuchgreen/howmuchcarbon";
+import {
+  getThingsFromQuery,
+  getTripFromQuery,
+} from "@howmuchgreen/howmuchcarbon";
 
-const macbookPro14 = howMuch("macbook pro 14").bestResult;
+const [macbookPro14] = getThingsFromQuery("macbook pro 14");
 console.log(macbookPro14.name); // 'MacBook Pro 14' 2021'
 console.log(macbookPro14.co2Eq.averageInGrams); // 271000
 console.log(macbookPro14.co2Eq.format()); // '271 kg'
 console.log(macbookPro14.sources); // ['https://www.apple.com/environment/…']
+
+const tripParisNewYork = getTripFromQuery("paris new york")?.trip!;
```

Comment:
Why could the trip be `null` or `undefined` ? `getTripFromQuery()` returns some result, I expect the `trip` property to be non-null, nope ?

Reply:
Oh I see it's because of the `getTripFromQuery("paris new york")?` that it can be null or undefined :)
So why not put an exclamation mark on `getTripFromQuery("paris new york")!` ?

Reply:
Actually, like you said below, I agree that `searchTrips` should return an array `TripWithScore[]`. I had the same idea, and I thought about changing the code to return multiple trips but it wasn't that easy so I decided to open the PR with only a single trip for now; however we can still return an array (with one element for now) to prepare for the future 👍 

So this part of the readme will de facto change.